### DISCUSSION TITLE
Manually disable the "Blocked by an object" sensors? (round bails)

### DISCUSSION THREAD

**Discussion by Chawpped** [Courseplay/Courseplay_FS22#1898] [Q&A] [answered]
tldr; Is there a way to tell my tractor to attempt to push past an object? Kind of how the vanilla AI helper does? My mulcher keeps stopping because it is scared of round cotton bails, even though he is fully capable of pushing right past them.

_I like to run my Cotton Harvester, and almost immediately follow it with my mulcher. Then just let the two of them work the field while I go do other work around the farm. Initially I had issues when the harvester was dumping cotton bails, and the mulching tractor would drive over them and flip himself. But after I equipped the mulching tractor with a frontloader and some pokers out front, it will actively push bails out of the way, instead of driving on top of them. So I had a perfect system, using the vanilla AI, except that it would regularly just... stop... in the middle of fields for no reason. Courseplay seems to be much more stable, and tractors will finish out an entire field without randomly quitting. I just need to be able to tell my mulcher that he's a big boy and he should at least try to push past an object for a bit before giving up._

![image](https://user-images.githubusercontent.com/29009983/189453420-8475dabf-4485-478a-ad43-e1cd5521069b.png)

**Comment by Tensuko** [comment, marked as answer]
We won't add a fix for setting for this.
If you have objects in your way, remove them as it should be done anyway.
Pushing things arround with Giants Engine is tricky anyway, you will just end up in another Problem.

Won't fix, sry.

**Reply by Chawpped** [reply]
I get it, thanks for the reply. I can stick with the default AI helper if I need to be pushing things around. It definitely does end up in hairy situations from time to time because of it. ;p
